src/attributes/initializer.js

```diff
@@ -19,24 +19,28 @@ const derivedInitializer = Object.assign({}, {
 });
 
 function initialize(attributes, schema, instance) {
-  instance.attributes = initializeWith(nativesInitializer, attributes, schema, instance);
-  instance.attributes = initializeWith(derivedInitializer, attributes, schema, instance);
+  let setAttributes = {};
```

Comment:
The name `setAttributes` sounds like it's a function, can we call it `initializedAttributes`?

Reply:
Sure, the"set" should have a past meaning on this context but I'm pretty sure `initialized` is a better option.